packages/appstore/src/components/loader/cfd-toggle-skeleton/cfdToggleSkeleton.scss

```diff
@@ -0,0 +1,23 @@
+@import '../skeletonLoader.scss';
+
+.cfd-toggle-skeleton-loader {
+    display: none;
+    flex-direction: column;
```

Comment:
Will flex-direction work if its display none ?

Reply:
same question as `display: none` totally hides the element

Reply:
No it's not needed since in mobile it's already block